internal/venice-common/src/test/java/com/linkedin/venice/blobtransfer/DaVinciBlobFinderTest.java

```diff
@@ -1,16 +1,11 @@
 package com.linkedin.venice.blobtransfer;
 
-import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-import static org.testng.AssertJUnit.assertEquals;
-import static org.testng.AssertJUnit.assertTrue;
+import static org.mockito.ArgumentMatchers.*;
```

Comment:
Can you fix this * import style by going to the IntelliJ project structure -> code style->Java and update the line to start * import to high number?